src/pages/index.js

```diff
@@ -135,10 +136,158 @@ class Home extends Component {
+                        },
+                      },
+
+                      [media.lessThan('medium')]: {
+                        display: 'inline-block',
+                        verticalAlign: 'top',
+                        marginLeft: 0,
+                        whiteSpace: 'normal',
+                        width: '75%',
+                        marginRight: 20,
+                        paddingBottom: 40,
+
+                        '&:first-of-type': {
+                          marginTop: 0,
+                        },
+                      },
+
+                      '& p': {
+                        lineHeight: 1.7,
+                      },
+                    }}>
+                    <h3
+                      css={{
+                        '&&': {
+                          // Make specificity higher than the site-wide h3 styles.
+                          color: colors.subtle,
+                          marginBottom: 20,
+                          paddingTop: 0,
+                          fontWeight: 300,
+                          fontSize: 20,
+
+                          [media.greaterThan('xlarge')]: {
+                            fontSize: 24,
+                            fontWeight: 200,
+                          },
+                        },
+                      }}>
+                      {column.title}
+                    </h3>
+                    <div dangerouslySetInnerHTML={{__html: column.content}} />
+                  </div>
+                ))}
+              </div>
+            </section>
+            <hr
+              css={{
+                height: 1,
+                marginBottom: -1,
+                border: 'none',
+                borderBottom: `1 solid ${colors.divider}`,
+              }}
+            />
+            <section css={sectionStyles}>
+              <div id="examples">
+                <div className="example">
+                  <h3>A Simple Component</h3>
+                  <p>
+                    React components implement a `render()` method that takes
+                    input data and returns what to display. This example uses an
+                    XML-like syntax called JSX. Input data that is passed into
+                    the component can be accessed by `render()` via
+                    `this.props`.
+                  </p>
+                  <p>
+                    <strong>
+                      JSX is optional and not required to use React.
+                    </strong>{' '}
+                    Try the{' '}
+                    <a href="http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&code_lz=MYGwhgzhAEASCmIQHsCy8pgOb2vAHgC7wB2AJjAErxjCEB0AwsgLYAOyJph0A3gFABIAE6ky8YQAoAlHyEj4hAK7CS0ADxkAlgDcAfAiTI-hABZaI9NsORtLJMC3gBfdQHpt-gNxDn_P_zUtIQAIgDyqPSi5BKS6oYo6Jg40A5OALwARCHwOlokmdBuegA00CzISiSEAHLI4tJeQA&debug=false&circleciRepo=&evaluate=false&lineWrap=false&presets=react&prettier=true&targets=&version=6.26.0">
```

Comment:
You know what might be cool? To auto-generate the URL for this from the actual code example (so the two will always stay in-sync). Babel's REPL just uses the [`lz-string` module](https://www.npmjs.com/package/lz-string) to [serialize and deserialize code](https://github.com/babel/website/blob/f34ae222d3f7edf70711e1c849f13418da95b4cb/js/repl/UriUtils.js#L22-L33) so we could copy it. 😁  It isn't likely to change because it's important to Babel to maintain backwards compatibility.

Reply:
If this link will be part of the markdown, how do we generate the URL for this? Do we generate after render and modify the `href` in the markup?